whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestServiceTestIntegration.java

```diff
@@ -6193,7 +6193,12 @@ public void mp_memebers_should_have_referenced_link_when_as_set() {
 
     }
 
-
+    @Test
+    public void bad_format_uri_then_not_found() {
```

Comment:
Only return bad request if the entity has an invalid format, otherwise return 404 if the object is not found